gtsfm/evaluation/metrics.py

```diff
@@ -0,0 +1,362 @@
+"""Class to store metrics computed in different GTSfM modules.
```

Comment:
can we add up here that if the full data is too large to be stored, we save only summary statistics like histogram bin counts or quartiles (e.g. for box plots)? It would make this file easier to understand.